check_command.go

```diff
@@ -67,7 +71,7 @@ func (c *CheckCommand) Run(request CheckRequest) ([]Version, error) {
 
 	for _, release := range filteredReleases {
 		if !upToLatest {
-			if *release.Draft {
+			if *release.Draft || *release.Prerelease {
```

Comment:
That was my initial thought, but https://github.com/google/go-github/blob/master/github/repos_releases.go#L27 uses ```Prerelease``` so all ```*release.Prerelease``` will fail.
As for the rest I followed the  the same style as draft. let me know if should still remove them.